app/controllers/api/v1/products_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V1::ProductsController < ApiController
+  # protect_from_forgery unless: -> { request.format.json? }
```

Comment:
this should be commented in, otherwise you will encounter more issues later when users play a role

Reply:
guys, just make sure we inherent ProductsController from ApiController not directly from ApplicationController since we have ApiController set up with "protect_from_forgery unless: -> { request.format.json? }", and this apiController inherents from ApplicationController.